### PR TITLE
Fix minor issues from Hugo

### PR DESCRIPTION
**Pull request by alxdavids**
Addresses #35

### CODE REVIEW

File: draft-sullivan-cfrg-oprf.md [hunk 1]
```diff
@@ -319,14 +319,15 @@ used k in the protocol using this commitment.
 # OPRF Protocol {#protocol}
 
 In this section we describe the OPRF protocol. Let GG be a prime-order additive
```
Comment (chris-wood) [inline draft-sullivan-cfrg-oprf.md]: GG is of prime order p, right? We should say so if that's the case.

File: draft-sullivan-cfrg-oprf.md [hunk 2]
```diff
@@ -460,7 +461,7 @@ the VOPRF analogues in {{voprf}}. We provide generic utility algorithms in
 ~~~
 Input:
 
- l: Some suitable choice of key-length (e.g. as described in {{NIST}}).
+ l: Some suitable choice of key-length (e.g. as described in [NIST]).
```
Comment (chris-wood) [inline draft-sullivan-cfrg-oprf.md]: Did {{NIST}} not work?

File: draft-sullivan-cfrg-oprf.md [hunk 3]
```diff
@@ -645,7 +646,7 @@ Output:
 
 Steps:
 
- 1. N := (1/r)Z
+ 1. N := (r^(-1)Z
```
Comment (chris-wood) [inline draft-sullivan-cfrg-oprf.md]:
```suggestion
 1. N := (r^(-1))Z
```

File: draft-sullivan-cfrg-oprf.md [hunk 4]
```diff
@@ -525,7 +526,7 @@ Output:
...
```
Comment (chris-wood) [inline draft-sullivan-cfrg-oprf.md]:
```suggestion
 1. N := (r^(-1))Z
```

File: draft-sullivan-cfrg-oprf.md [hunk 5]
```diff
@@ -816,6 +819,11 @@ Steps:
  5. Output D := (c, s)
 ~~~
 
+We note here that it is essential that a different r value is used for every
+invocation. If this is not done, then this may leak the key k in a similar
+fashion as is possible in Schnorr/(EC)DSA scenarios where frash randomness is
```
Comment (chris-wood) [inline draft-sullivan-cfrg-oprf.md]:
```suggestion
fashion as is possible in Schnorr or (EC)DSA scenarios where fresh randomness is
```

File: draft-sullivan-cfrg-oprf.md [hunk 6]
```diff
@@ -702,12 +703,14 @@ values. For example, computing rH_1(x) is an example of multiplicative blinding.
 A valid way of computing additive blinding would be to instead compute
 H_1(x)+rG, where G is the common generator for the group.
 
-If the client preprocesses values of the form rG, then computing H_1(x)+rG is
-more efficient than computing rH_1(x) (one addition against log_2(r)).
-Therefore, it may be advantageous to define the OPRF and VOPRF protocols using
-additive blinding rather than multiplicative blinding. In fact the only
-algorithms that need to change are OPRF_Blind and OPRF_Unblind (and similarly
-for the VOPRF variants).
+By pre-processing tables of blinded scalar multiplications for the specific
```
Comment (chris-wood) [inline draft-sullivan-cfrg-oprf.md]: I wonder if this is a choice we should specify. It seems use of a different blinding technique is an entirely separate protocol. Maybe we ought to (in a future issue) make the blinding choice a parameter and put it behind a parameterized function, e.g., `Blind()`?

File: draft-sullivan-cfrg-oprf.md (hunk 2 above)
Comment (alxdavids) [reply draft-sullivan-cfrg-oprf.md]: No, because it's in a ~~~ block I believe

File: draft-sullivan-cfrg-oprf.md (hunk 6 above)
Comment (alxdavids) [reply draft-sullivan-cfrg-oprf.md]: I agree, I think this would be a good change to make for a future version.

Comment (alxdavids) [reply draft-sullivan-cfrg-oprf.md]: I'll make a separate issue to track it after the submission today

**Comment by chris-wood** [comment]
@alxdavids can you resolve the conflicts? Feel free to merge when done!

**Pull request merged**